elixir/apps/web/lib/web/live/sites/edit.ex

```diff
@@ -28,12 +28,50 @@ defmodule Web.Sites.Edit do
           <.form for={@form} phx-change={:change} phx-submit={:submit}>
             <div class="grid gap-4 mb-4 sm:grid-cols-1 sm:gap-6 sm:mb-6">
               <div>
-                <.input
-                  label="Name Prefix"
-                  field={@form[:name]}
-                  placeholder="Name of this Site"
-                  required
-                />
+                <.input label="Name" field={@form[:name]} placeholder="Name of this Site" required />
+              </div>
+              <div>
+                <p class="text-lg text-slate-900 mb-2">
+                  Data Routing -
+                  <a
+                    class={[link_style(), "text-sm"]}
+                    href="https://www.firezone.dev/kb"
+                    target="_blank"
+                  >
+                    Read about routing in Firezone
+                  </a>
+                </p>
+                <div>
+                  <div>
+                    <.input
+                      id="routing-option-managed"
+                      type="radio"
+                      field={@form[:routing]}
+                      value="managed"
+                      label="Firezone Managed Relays"
+                      checked={@form[:routing].value == :managed}
+                      required
+                    />
+                    <p class="ml-6 mb-4 text-sm text-slate-500 dark:text-slate-400">
+                      Firezone will route connections through our managed Relays only if a direct connection to a Gateway is not possible.
+                      Firezone can never decrypt the contents of your traffic.
+                    </p>
+                  </div>
+                  <div>
+                    <.input
+                      id="routing-option-stun-only"
+                      type="radio"
+                      field={@form[:routing]}
+                      value="stun_only"
+                      label="Direct Only"
+                      checked={@form[:routing].value == :stun_only}
+                      required
+                    />
+                    <p class="ml-6 mb-4 text-sm text-gray-500 dark:text-gray-400">
+                      Firezone will enforce direct connections to all Gateways in this Site.  This could cause connectivity issues in rare cases.
```

Comment:
(nit) HTML will collapse this anyway
```suggestion
                      Firezone will enforce direct connections to all Gateways in this Site. This could cause connectivity issues in rare cases.
```

Reply:
Good catch.  Putting two spaces after a period is just ingrained in my muscle memory so deep.